src/index.js

```diff
@@ -28,3 +29,5 @@ ReactDOM.render(
   </div>,
   document.getElementById('root')
 );
+
+store.dispatch(ActionCreators.initialize());
```

Comment:
> Generic action that can handle any sort of initialization details of our app.

LOL. Basically, any work that we need to do when the application starts up can be handled by this action. So things like updating the user's local album links from the database, fetching all preview images, etc. etc.